microsoft/testsuites/core/storage.py

```diff
@@ -70,9 +70,19 @@ def verify_disks_device_timeout_setting(
         disks = node.features[Disk].get_all_disks()
         root_device_timeout_from_waagent = node.tools[Waagent].get_root_device_timeout()
         for disk in disks:
-            device_timeout_from_distro = int(
-                node.tools[Cat].run(f"/sys/block/{disk}/device/timeout").stdout
-            )
+            timeout = 60
+            timer = create_timer()
+            while timeout > timer.elapsed(False):
+                device_timeout_from_distro = int(
+                    node.tools[Cat]
+                    .run(f"/sys/block/{disk}/device/timeout", force_run=True)
+                    .stdout
+                )
+                if root_device_timeout_from_waagent == device_timeout_from_distro:
+                    break
+                else:
+                    time.sleep(1)
+                    continue
```

Comment:
It doesn't need `continue` at the end of while loop.

Reply:
update